src/overlay/Peer.h

```diff
@@ -500,4 +491,17 @@ class Peer : public std::enable_shared_from_this<Peer>,
         }
     }
 };
+
+class MsgCapacityTracker : private NonMovableOrCopyable
```

Comment:
Can you please add a class-level comment explaining what a `MsgCapacityTracker` is?

Reply:
Added.

Reply:
Given the elevated role of `MsgCapacityTracker` in this PR -- as _the_ way messages get passed around now -- I wonder if it ought to get a more central-sounding name like `CapacityTrackedMessage` or something?